module-4/backend/week_one.md

```diff
@@ -1,19 +1,50 @@
 ## Week One - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+1. What's the most useful thing you learned from completing the intermission week work?
 
-Note: When you're done, submit a PR. 
+  The most important thing that I learned from completing the intermission week work is JavaScript syntax.
 
-1. What's the most useful thing you learned from completing the intermission week work?
 2. What are some tools to help debug JavaScript code?
+
+  Some tools to help me debug JavaScript are pry, console.log, and the console in your browser.
+
 3. What are some tools you need in order to unit test your JavaScript?
+
+  You need mocha and chai to test your JavaScript.
+
 4. What is the syntax for invoking a function?
+
+  function name(){
+
+  }
+
+  name()
+
 5. What's `this` in JavaScript?
+
+  `this` is the outer object of the current object that you are in.
```

Comment:
Could you give a little more detail here? What is `this` when you are inside an object, function, or global?